packages/components/src/popover/index.js

```diff
@@ -374,59 +373,42 @@ const Popover = (
 		return autoUpdate(
 			resultingReferenceRef,
 			refs.floating.current,
-			update
+			update,
+			{
+				animationFrame: true,
```

Comment:
I noticed that you did some follow-up work to this, so I was wondering if this is still necessary. The reason I'm asking is because this line is a source of a large impact on performance degradation potentially.

It now happens in the "select" and "inserter open" metrics after this PR https://github.com/WordPress/gutenberg/pull/42722 but it can also happen I guess for other interactions when there's a big list of popovers.

So can we remove it or is it still necessary? Are there alternatives?

Reply:
I guess this can be opt-in (or opt-out). However, I think it's generally a bad practice to render a large list of popovers at the same time. The performance regression in #42722 is a bit extreme that we render 1000 empty paragraphs in the test, but I do think we can refactor it in a follow-up to improve this.

From the [floating-ui](https://floating-ui.com/docs/autoUpdate#animationframe) doc:

> This is optimized for performance but can still be costly.

> This option should be used sparingly...

I guess it makes sense to make it an opt-in in the next major release?

Reply:
This was introduced mainly to fix https://github.com/WordPress/gutenberg/issues/42725, I expect that bug (and any other nested popover) to happen again if we undo this change

While I agree that calling the `getBoundingClientRect` function every animation frame is not ideal, I would also argue that the main reason while this can be a noticeable regression is that we're rendering hundreds of popovers at the same time.

Reply:
It turned out that the main issue for the performance regression is not this, so I'm ok not doing anything here and keeping it as but we should keep an eye on this any way as mentioned on their docs.